### PR TITLE
Fix grouped elements angle calculation issue

### PR DESCRIPTION
**Pull request by net5566**

### CODE REVIEW

File: public/js/lib/svgeditor/svgcanvas.js [hunk 1]
```diff
@@ -10378,7 +10378,7 @@ define([
                         var coldc = svgedit.math.transformPoint(cbox.x + cbox.width / 2, cbox.y + cbox.height / 2, ceqm);
 
                         // sum group and child's angles
-                        var sangle = gangle + cangle;
+                        var sangle = (gangle + cangle) % 360;
```
Comment (dkdkwizard) [inline public/js/lib/svgeditor/svgcanvas.js:10381]: Angle needs to be bounded in -180 ~ 180.
```js
if (sangle > 180) {
  sangle -= 180;
} else if (sangle < -180) {
  sangle += 180;
}
```

**Review by dkdkwizard — CHANGES_REQUESTED** [review]
Tiny fix needed.

File: public/js/lib/svgeditor/svgcanvas.js [hunk 2]
```diff
@@ -10366,7 +10366,13 @@ define([
             var coldc = svgedit.math.transformPoint(cbox.x + cbox.width / 2, cbox.y + cbox.height / 2, ceqm);
 
             // sum group and child's angles
-            var sangle = gangle + cangle;
+            var sangle = (gangle + cangle) % 360;
+
+            if (sangle > 180) {
+              sangle -= 180;
+            } else if (sangle < -180) {
+              sangle += 180;
```
Comment (dkdkwizard) [inline public/js/lib/svgeditor/svgcanvas.js:10374]: Should be 360.
Sorry, my bad.

**Review by dkdkwizard — APPROVED** [review]
+2

**Review by ChenYanting — APPROVED** [review]
LGTM

**Pull request merged**